File: MasgedParentMobileAPI/Teacher/Entities/HeroSlide.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace MasgedTeacherMobileAPI.Entities;

[Table("HeroSlide")]
public class HeroSlide
{
    public int Id { get; set; }

    [StringLength(500)]
    public string? ImageUrl { get; set; }

    public int SortOrder { get; set; }

    public DateTime CreatedAt { get; set; }
}

File: AdminPanelUI/src/types/heroSlide.ts
export interface HeroSlideListItem {

  id: number

  imageUrl: string | null

  sortOrder: number

}



export interface HeroSlide {

  id: number

  imageUrl: string | null

  sortOrder: number

}



export interface SaveHeroSlidePayload {

  sortOrder: number

  imageFiles: File[]

}

